invokeai/frontend/web/src/features/modelManagerV2/subpanels/AddModelPanel/HuggingFaceFolder/HuggingFaceResults.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Button,
+  Divider,
+  Flex,
+  Heading,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+} from '@invoke-ai/ui-library';
+import { useAppDispatch } from 'app/store/storeHooks';
+import ScrollableContent from 'common/components/OverlayScrollbars/ScrollableContent';
+import { addToast } from 'features/system/store/systemSlice';
+import { makeToast } from 'features/system/util/makeToast';
+import type { ChangeEventHandler } from 'react';
+import { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PiXBold } from 'react-icons/pi';
+import { useInstallModelMutation } from 'services/api/endpoints/models';
+
+import { HuggingFaceResultItem } from './HuggingFaceResultItem';
+
+type HuggingFaceResultsProps = {
+  results: string[];
+};
+
+export const HuggingFaceResults = ({ results }: HuggingFaceResultsProps) => {
+  const { t } = useTranslation();
+  const [searchTerm, setSearchTerm] = useState('');
+  const dispatch = useAppDispatch();
+
+  const [installModel] = useInstallModelMutation();
+
+  const handleSearch: ChangeEventHandler<HTMLInputElement> = useCallback((e) => {
```

Comment:
It doesn't seem that searching actually does anything here

Reply:
Good catch, forgot to finish that!